src/NzbDrone.Core/Parser/QualityParser.cs

```diff
@@ -70,6 +73,16 @@ public static QualityModel ParseQuality(string name)
             var resolution = ParseResolution(normalizedName);
             var codecRegex = CodecRegex.Match(normalizedName);
 
+            if (RemuxRegex.IsMatch(normalizedName) && sourceMatch?.Groups["webdl"].Success != true && sourceMatch?.Groups["hdtv"].Success != true)
```

Comment:
There are a lot of other sources that could match here that should be excluded, specifying a bluray source would mean it'd match only when bluray and not have to exclude every other source that could be added.